lib/generators/react_on_rails/directory_generator.rb

```diff
@@ -0,0 +1,37 @@
+require "rails/generators"
+
+module ReactOnRails
+  module Generators
+    class DirectoryGenerator < Rails::Generators::Base
+      source_root File.expand_path("../templates", __FILE__)
+
+      def create_react_and_redux_dir
```

Comment:
I think it is okay if the project is 'opinionated' on Redux (and willing to switch to the most valuable Flux implementation in the future if it switches).  Have an option, to 'opt-out' of having redux as the default.  But it is okay if the generator provides 'react_on_rails' opinion on the best seed/starter setup.